tinygrad/runtime/ops_python.py

```diff
@@ -67,12 +67,12 @@ def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tup
         assert dtype is not None, f"{uop} is missing a dtype"
         dl[i] = dtype
         if uop is Ops.DEFINE_GLOBAL:
-          assert dtype.fmt is not None
-          ul[i] = [pbufs.pop(0).cast(dtype.fmt)] * warp_size
+          assert (dtype.fmt is not None)
+          ul[i] = [pbufs.pop(0).cast(cast(LegacyFmtStrType, dtype.fmt))] * warp_size
```

Comment:
The stubs for memoryview cast have not yet been updated for half https://github.com/python/typeshed/blob/e1c74f08f12b8bec6afe266951f0756dc1b43ebe/stdlib/builtins.pyi#L872